python/cudf/cudf/core/series.py

```diff
@@ -1543,8 +1542,18 @@ def _binaryop(
                     rhs = rhs.fillna(fill_value)
 
         outcol = lhs._column.binary_operator(fn, rhs, reflect=reflect)
-        result = lhs._copy_construct(data=outcol, name=result_name)
-        return result
+
+        # Get the appropriate name for output operations involving two objects that
+        # are a mix of pandas and cudf Series and Index. All of these object
```

Comment:
Looks like this comment is incomplete :)

Reply:
No idea what you're talking about 🙃